circle/docker-release-image.sh

```diff
@@ -48,18 +48,31 @@ RELEASE_SERIES="${TARGET_BRANCH}"
 if [ -n "${VARIANT}" ]; then
   RELEASE_SERIES+="-${VARIANT}"
 fi
-CACHE_TAG=${RELEASE_SERIES}
 
+CACHE_TAG=${RELEASE_SERIES}
 if [ "${IS_DEFAULT_DISTRO}" == 0 ] ; then
-  RELEASE_SERIES+="/${DISTRO}"
   CACHE_TAG+="-${DISTRO}"
 fi
-TAGS_TO_UPDATE+=($CACHE_TAG $IMAGE_TAG $ROLLING_IMAGE_TAG)
 
-if [[ "${IS_DEFAULT_DISTRO}" == 1 && $RELEASE_SERIES == $LATEST_STABLE ]]; then
-  [[ $LATEST_TAG_SOURCE == "LATEST_STABLE" ]] && TAGS_TO_UPDATE+=('latest')
+# This case is specific for debian-8 now
+# TODO(jdrios) remove once it is fully deprecated
+if [[ "${DISTRO}" != "debian-8" ]]; then
+  RELEASE_SERIES+="/${DISTRO}"
+  LATEST_STABLE+="/${DISTRO}"
```

Comment:
Are you sure this is correct? `LATEST_STABLE` is set externally by the `circle.yml` file, so I wouldn't modify it to avoid possible confusions. Use a new variable if you need it